plugins/versionpress/src/Initialization/ReplacerMethods.src.php

```diff
@@ -91,15 +96,16 @@ public function delete($table, $where, $where_format = null) {
         return $r;
     }
 
-    public function query($query) {
+    public function query($query)
+    {
         global $versionPressContainer;
         if ($versionPressContainer == null || $this->vp_use_original_query_method) {
             return $this->__wp_query($query);
         }
 
         $r = null;
 
-        /** @var SqlQueryParser $sqlQueryParser */
+        /** @var \VersionPress\Database\SqlQueryParser $sqlQueryParser */
```

Comment:
It's not important as it's just a comment. But it wasn't valid class in the context of replaced `wp-db.php`.